_data/team/roles/mdakit_registry.yml

```diff
@@ -3,7 +3,7 @@ tasks:
   - Managing submissions to MDAKits registry
   - Managing manual review process of new MDAKits
   - Managing automated test and badge infrastructure of all MDAKits
-  - Manage helping with MDAKits who need assistance
+  - Offer input on MDAKit issues where possible
```

Comment:
I'm starting to think this shouldn't be the team's responsibility at all. We can help folks during the review process and if they fail in a way that is picked up with the registry, but we shouldn't be stewards of MDAKits outside the registry.